Dockerfile

```
@@ -0,0 +1,30 @@
+# Use a minimal base image with GCC installed
+FROM alpine:3.18
+
+# Install necessary build tools
+RUN apk add --no-cache \
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)